### PR TITLE
Add support of RTL UI in login pages

### LINKED ISSUE #29974: Add support of RTL UI in login pages
### Description

Support RTL UI for login pages.

Part of #17904

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by Eng-Fouad**
Resolves #29974

### CODE REVIEW

File: services/src/main/java/org/keycloak/theme/beans/LocaleBean.java
Comment (jonkoops) [inline services/src/main/java/org/keycloak/theme/beans/LocaleBean.java:69]: Let's add a bit of JavaDoc for those not familiar with the RTL terminology.

File: themes/src/main/resources/theme/base/login/login-config-totp.ftl
Comment (jonkoops) [inline themes/src/main/resources/theme/base/login/login-config-totp.ftl:60]: Adding `dir` attributes selectively here and there seems like a very error prone way to do this, as it is likely to get our of sync of simply be forgotten. Would setting this globally not be enough?

Comment (Eng-Fouad) [reply themes/src/main/resources/theme/base/login/login-config-totp.ftl:60]: @jonkoops username and password are mostly in English, so that's why I chose to make those text fields always LTR.

**Review by jonkoops — APPROVED** [review]
LGTM

**Comment by Eng-Fouad** [comment]
@jonkoops I am not sure why `Keycloak CI / Status Check - Keycloak CI` failed.

**Comment by jonkoops** [comment]
Probably just needs a kick, let me re-start it.

**Comment by Eng-Fouad** [comment]
@jonkoops Does this PR need another reviewer before merge?

**Review by ssilvert — APPROVED** [review]
LGTM

**Pull request merged**
